src/reducer/data-reducer.js

```diff
@@ -0,0 +1,10 @@
+export const dataReducer = (dataState, action) => {
```

Comment:
Suggestion: destructure action here like:
`export const dataReducer = (dataState, {type, payload})`

Reply:
nice suggestion but will do it next time!